code/controllers/subsystem/shuttle.dm

```diff
@@ -1,4 +1,4 @@
-#define HIGHLIGHT_DYNAMIC_TRANSIT 1
+#define HIGHLIGHT_DYNAMIC_TRANSIT 0
```

Comment:
Turning this off since it's a debug option which should probably not be on by default.